src/components/AddCard/AddCard.jsx

```diff
@@ -0,0 +1,14 @@
+import styles from './AddCard.module.css';
+import Plus from '../../assets/images/plus.svg'
+
+
+function AddCard() {
```

Comment:
Perhaps this could more accurately be made `AddCardButton`? Up to you

Reply:
yup changed